.github/workflows/test.yml

```diff
@@ -172,40 +172,35 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Download graphviz source
-      run: Invoke-WebRequest -Uri "https://gitlab.com/graphviz/graphviz/-/package_files/6105690/download" -OutFile "C:\Temp\development_windows_10_cmake_Release_x64_graphviz-install-2.44.2_dev.20210115.1523-win64.exe"
+      run: |
+        Invoke-WebRequest `
+          -Uri "https://gitlab.com/graphviz/graphviz/-/package_files/6105690/download" `
+          -OutFile "C:\Temp\graphviz-install-2.44.2_dev.20210115.1523-win64.exe"
 
     - name: Install exe
-      run: Start-Process -Wait -FilePath "C:\Temp\development_windows_10_cmake_Release_x64_graphviz-install-2.44.2_dev.20210115.1523-win64.exe" -ArgumentList '/S' -PassThru
-
-    - name: List
-      run: | 
-        Get-ChildItem "C:\Program Files\" 
-        Get-ChildItem "C:\Program Files\Graphviz\" 
-        Get-ChildItem "C:\Program Files\Graphviz\bin\" 
-        Get-ChildItem "C:\Program Files\Graphviz\include\" 
-        Get-ChildItem "C:\Program Files\Graphviz\lib\" 
```

Comment:
While not necessary for the build, it might actually be useful for debugging in the future (or demonstrating the Graphviz installation structure) to leave this step or something like it in. That way, if something unexpected happens with the Graphviz installation, we might be able to catch it in the logs.

Reply:
I added back ``Get-ChildItem "C:\Program Files\Graphviz\bin\"`` so we remember the syntax and it should help us notice if things aren't in "C:\Program Files\Graphviz\".